### PR TITLE
Adding clickhouse on data integrations destination

### PR DESCRIPTION
**Pull request by Luishfs**
# Summary
adding clickhouse on data integrations destination

# Tests
Tested using API source  on clickhouse destination

Screenshot from 2023-07-18 01-24-11

Tested using PostgreSQL on clickhouse destination
PostgreSQL data was created from Salesforce Source data, full table schema:

Screenshot from 2023-07-24 15-20-19
Screenshot from 2023-07-24 15-20-34

Clickhouse query example (removed default visualization for readability purposes) :
Screenshot from 2023-07-24 15-21-12

cc:
@wangxiaoyou1993

### CODE REVIEW

**Comment by wangxiaoyou1993** [comment]
How does the Clickhouse destination write state? Did you test incremental sync?

**Comment by Luishfs** [comment]
> How does the Clickhouse destination write state? Did you test incremental sync?

Added state support on [497ca3a](https://github.com/mage-ai/mage-ai/pull/3005/commits/497ca3ac4e310c8d483fdbe118f9c1afb98822f0)

File: mage_integrations/mage_integrations/destinations/clickhouse/templates/config.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "sqlalchemy_url": ""
+}
```
Comment (wangxiaoyou1993) [inline mage_integrations/mage_integrations/destinations/clickhouse/templates/config.json:3]: nit: add empty line

File: requirements.txt [hunk 2]
```diff
@@ -87,9 +87,9 @@ pika==1.3.1
 pymongo==4.3.3
 
 # DBT
-dbt-bigquery==1.4.0
-dbt-core==1.4.0
-dbt-postgres==1.4.0
+dbt-bigquery==1.4.3
```
Comment (wangxiaoyou1993) [inline requirements.txt:90]: You also need to update the dbt packge version in setup.py
* https://github.com/mage-ai/mage-ai/blob/master/setup.py#L121
* https://github.com/mage-ai/mage-ai/blob/master/setup.py#L64

File: mage_ai/frontend/interfaces/IntegrationSourceType.ts [hunk 3]
```diff
@@ -137,6 +137,7 @@ export enum IntegrationSourceEnum {
 
 export enum IntegrationDestinationEnum {
   BIGQUERY = 'bigquery',
+  CLICKHOUSE = 'clickhouse',
```
Comment (wangxiaoyou1993) [inline mage_ai/frontend/interfaces/IntegrationSourceType.ts:140]: no need to update frontend code here. we get destination list from backend.

**Comment by wangxiaoyou1993** [comment]
What data type have you tested? Could you include more testing details in your PR description.

File: mage_ai/frontend/interfaces/IntegrationSourceType.ts (hunk 3 above)
Comment (Luishfs) [reply mage_ai/frontend/interfaces/IntegrationSourceType.ts]: removed on d27c638831d274c104381dcc25fde217b37e0a86

File: requirements.txt (hunk 2 above)
Comment (Luishfs) [reply requirements.txt:90]: updated on d27c638831d274c104381dcc25fde217b37e0a86

File: mage_integrations/mage_integrations/destinations/clickhouse/templates/config.json (hunk 1 above)
Comment (Luishfs) [reply mage_integrations/mage_integrations/destinations/clickhouse/templates/config.json]: added on d27c638831d274c104381dcc25fde217b37e0a86

**Review by wangxiaoyou1993 — APPROVED** [review]

**Pull request merged**